src/main/resources/graphics/scenery/volumes/BDVVolume.frag

```diff
@@ -101,14 +101,14 @@ void main()
 	float tnear = 1, tfar = 0, tmax = getMaxDepth( depthUV );
 	float n, f;
 
-	// $repeat:{vis,intersectBoundingBox|
+	// $repeat:{vis,skip,intersectBoundingBox|
 	bool vis = false;
 	intersectBoundingBox( wfront, wback, n, f );
 	f = min( tmax, f );
-	if ( n < f )
+	if ( n < f && skip == 0)
```

Comment:
I forgot what we did this skip thing. During the merge there was something from main that looks like it would do the same as skip but it is not present at this position in the code. I was called scenegraphvisibility or something like that. At the current moment skip is not set in the kotlin code but volumes still run. Though this bvv flickering is present.